app/services/listing_index_service/data_types.rb

```diff
@@ -17,7 +17,7 @@ module ListingIndexService::DataTypes
     [:per_page, :to_integer, :mandatory, gte: 1],
     [:keywords, :string, :optional],
     [:categories, :array, :optional],
-    [:listing_shape_id, :fixnum, :optional],
+    [:listing_shape_id, :array, :optional],
```

Comment:
@rap1ds  I thought of renaming to plural too, but with quick test passing that to search engine didn’t seem to work. I guess as sphinx index has listing_shape id it matches there. And changing it to plural wouldn’t make sense in the index as one listing only has one id. And if the param would be listing_shapre_id in the app it would need to be converted to listing_shape_id before passing to sphinx. 

So with my current understanding would be simpler to call it listing_shape_id all the time..